framework/src/DriverFramework.cpp

```diff
@@ -602,36 +601,34 @@ void HRTWorkQueue::process(void)
 					item->updateStats(now);
 
 					// reschedule work
-					item->m_queue_time = offsetTime();
+					item->m_queue_time += item->m_delay_usec;
```

Comment:
m_queue_time used to be the offset time that the work item was (re)queued. Now it is being set to the delay interval.